file/src/main/scala/akka/stream/alpakka/file/scaladsl/Directory.scala

```diff
@@ -42,4 +43,26 @@ object Directory {
     StreamConverters.fromJavaStream(factory)
   }
 
+  /**
+   * Create local directories, including any parent directories.
+   */
+  def mkdirs(): Flow[Path, Path, NotUsed] =
+    Flow[Path]
+      .map(Files.createDirectories(_))
```

Comment:
I know filesystem ops are synchronous, but I wonder if this could be a bottleneck when we want to create lots of directories.  Would it make sense to offer an async version or leave it as a task to the user?

Reply:
This will run on Akka's IO Dispatcher so it is async. What kind of async do you have in mind?

Reply:
I mean should we use `mapAsync` with some default parallelism when creating directories?

Reply:
I don't think that's useful. It would require wrapping it in a `Future` which would need an execution context.
In the end, the file system is a giant shared resource with a big semaphore somewhere. Creating directories in parallel might even show strange behaviour.